### PR TITLE
move collection pr quickstart over

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Moves over another collection docs file from community-docs. Provides a hand-holding approach to introduce new contributors to an existing collection how to create a good PR.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/community/create_pr_quick_start.rst
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by samccann** [comment]
replaces this file -https://github.com/ansible/community-docs/blob/main/contribution_to_project.rst

File: docs/docsite/rst/community/collection_development_process.rst [hunk 1]
```diff
@@ -39,9 +39,9 @@ Here's an overview of the PR lifecycle:
 Making your PR merge-worthy
 ===========================
 
-We do not merge every PR. Here are some tips for making your PR useful, attractive, and merge-worthy.
+We do not merge every PR. See :ref:`collection_quickstart` for  tips to make your PR useful, attractive, and merge-worthy.
```
Comment (Andersson007) [inline docs/docsite/rst/community/collection_development_process.rst:42]:
```suggestion
We do not merge every PR. See :ref:`collection_quickstart` for  tips to make your PR useful, attractive, and merge-worthy.
```
How about (also) adding this ref behind "Contributor opens a PR" in the paragraph above?
Later, when moving other docs, we could put the links to them in the corresponding items, for example, a ref to `review checklist` in "Developers, maintainers, community review the PR", etc. What do you think?

Comment (samccann) [reply docs/docsite/rst/community/collection_development_process.rst:42]: sounds like a good plan.

File: docs/docsite/rst/community/create_pr_quick_start.rst [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+Test your changes
+=================
+
+1. Run ``flake8`` against a changed file:
+
+  .. code-block:: bash
+
+    $ flake8 path/to/changed_file.py
+
+  It is worth installing (``pip install flake8``, or install the corresponding package on your operating system) and running ``flake8`` against the changed file(s) first. It shows unused imports, which is not shown by sanity tests (see the next step), as well as other common issues.
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/create_pr_quick_start.rst]: Would it make sense to put this at the beginning of step 1, ie mention how to install `flake8` before giving the command?

File: docs/docsite/rst/community/create_pr_quick_start.rst [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  If they failed, look at the output carefully - it is informative and helps to identify a problem line quickly.
+  Sanity failings usually relate to incorrect code and documentation formatting.
+
+3. Run integration tests:
+
+  .. code-block:: bash
+
+    $ ansible-test integration name_of_test_subdirectory --docker -v
+
+  For example, if the tests files you changed are stored in ``tests/integration/targets/test_mysql_user/``, the command is:
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/create_pr_quick_start.rst]: Should this be test files?

File: docs/docsite/rst/community/create_pr_quick_start.rst [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+=====================
+
+1. Commit your changes with an informative but short commit message:
+
+  .. code-block:: bash
+
+    $ git add /path/to/changed/file
+    $ git commit -m "module_name_you_fixed: fix crash when ..."
+
+2. Push the branch to the ``origin`` (your fork):
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/create_pr_quick_start.rst]: origin repository / origin remote?

File: docs/docsite/rst/community/create_pr_quick_start.rst [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+  GitHub is tracking your fork, so it should see the new branch in it and automatically offer  to create a pull request. Sometimes GitHub does not do it, and you should click the :guilabel:`New pull request` button yourself. Then choose :guilabel:`compare across forks` under the :guilabel:`Compare changes` title.
+
+5.Choose your repository and the new branch you pushed in the right drop-down list. Confirm.
+
+  a. Fill out the pull request template with all information you want to mention.
+
+  b. Put ``Fixes + link to the issue`` in the pull request's description.
+
+  c. Put ``[WIP] + short description`` in the pull request's title. It's often a good idea to mention the name of the module/plugin you are modifying at the beginning of the description.
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/create_pr_quick_start.rst]: Suggestion: delete "It's often a good idea to", and start the sentence with "Mention the name"

File: docs/docsite/rst/community/create_pr_quick_start.rst [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+
+      $ansible-test sanity changelogs/fragments/ --docker -v
+
+
+  b. If the tests passed, commit and push the changes:
+
+    .. code-block:: bash
+
+      $ git add changelogs/fragments/myfragment.yml
+        $ git commit -m "Add changelog fragment"
```
Comment (ariordan-redhat) [inline docs/docsite/rst/community/create_pr_quick_start.rst]: Is the indentation necessary here?

**Review by ariordan-redhat — COMMENTED** [review]
A few minor suggestions, but LGTM.

**Comment by Andersson007** [comment]
@samccann @ariordan-redhat thank you!

**Pull request merged**